concent_api/core/admin.py

```diff
@@ -67,21 +70,21 @@ class SubtaskAdmin(ModelAdminReadOnlyMixin, admin.ModelAdmin):
     ]
 
     @classmethod
-    def get_provider_public_key(cls, obj):
+    def get_provider_public_key(cls, obj: Subtask) -> str:
```

Comment:
if it is Subtask instance, maybe change `obj -> subtask`. It is in few places

Reply:
:+1: done